assignments/closure.js

```diff
@@ -2,14 +2,50 @@
 // Write a simple closure of your own creation.  Keep it simple!
 
 
-// ==== Challenge 2: Create a counter function ====
-const counter = () => {
-  // Return a function that when invoked increments and returns a counter variable.
+function sayMeow(meow) {
+  const myMeow = meow;
+  console.log(`I am a cat and I like to ${myMeow}`);
+
+  function myHairball() {
+    const hairBalled = "*cough* *wheeze*";
+    console.log(`My ${myMeow}s can be nice until I get a hairball.`);
+
+    function nowChoke() {
+      const nineLives = "Revive!"
+      console.log(`${myMeow}ing , oh god ${hairBalled} , noooooo.` )
+    };
+    nowChoke();
+  };
+  myHairball();
 };
+
+sayMeow("Purrrrrr");
+
```

Comment:
nice creation! one thing is missing
```suggestion
// Return a function that when invoked increments and returns a counter variable.
```